docs/tutorial.rst

```diff
@@ -40,6 +40,7 @@ Inference on random graphs depends on low-dimensional Euclidean representation o
    :maxdepth: 1
    
    tutorials/embedding/AdjacencySpectralEmbed
+   tutorials/embedding/OutOfSampleEmbed
```

Comment:
Is this confusing since this isn't the name of a class? just a q, what do you think?